### DISCUSSION TITLE
Command line admin utility

### DISCUSSION THREAD

**Discussion by D4V3M0NK** [Ylianst/MeshCentral#6195] [General]
Greetings to all - I trust that all are well and healthy.

I've spent the last hour or so looking back through old discussions and bugs and I can't seem to find what I know is out there: and that is, the command line utility that can be run on the MC server to (amongst other things) set up an admin account, outside of the UI.

Can anyone give me some pointers as to where that utility is, and if there are any online references to check out what it is capable of?

Thanks in advance,

D

**Comment by si458** [comment]
`meshctrl.js` it's a nodejs file that's included with the meshcentral :)

**Reply by D4V3M0NK** [reply]
@si458 Hey Simon,
Thank you! That's where it is!!

For reference:
`node node_modules/meshcentral/meshctrl` will show all options

**Reply by silversword411** [reply]
Don't forget to support for the support provided 
https://github.com/si458
